src/templates/antora/antora.converter.ts

```diff
@@ -79,19 +76,27 @@ const cleanup = (str: string): string => {
   return filters.reduce((acc, filter) => filter(acc), str);
 };
 
-const getNameFromFullName = (name: string): string =>
-  name.split('.').slice(-1).join('');
-
 const getFilePathFromFullName = (name: string): string => {
   const filename = name.toLowerCase() === 'tinymce' ? 'tinymce.root' : name.toLowerCase();
-  return basePath + filename + '.adoc';
+  return 'apis/' + filename + '.adoc';
 };
 
-const generateTypeXref = (type: string): string =>
-  type.includes('tinymce', 0) ? 'xref:' + getFilePathFromFullName(type) + '[' + getNameFromFullName(type) + ']' : type;
+const getFilePathFromFullNameLegacy = (name: string): string => {
+  const folder = name.split('.').slice(0, -1).join('.') + '/';
+  const filename = name.toLowerCase() === 'tinymce' ? 'tinymce/root_tinymce' : name.toLowerCase();
+  return 'api/' + folder + filename + '.adoc';
+};
 
-const generateDefinedByXref = (definedBy: string): string =>
-  'xref:' + getFilePathFromFullName(definedBy) + '[' + getNameFromFullName(definedBy) + ']';
+const getFilePath = (structure: string): Function =>
```

Comment:
This return type is incorrect... you're losing type information by doing this.
```suggestion
const getFilePath = (structure: string): (name: string) => string =>
```